web/src/main/java/com/futurewei/alcor/web/entity/port/LocalLinkInformation.java

```diff
@@ -0,0 +1,12 @@
+package com.futurewei.alcor.web.entity.port;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LocalLinkInformation {
+    @JsonProperty("port_id")
+    private  String port_id="Ethernet3/1";
```

Comment:
is it required to fill in these fields? They appear to be information from physical networks which don't apply to our case. 

See if we should assign ""